doc/api/cli.md

```diff
@@ -25,19 +25,19 @@ _For more info about `node debug`, please see the [debugger][] documentation._
 
 ### `-v`, `--version`
 
-Print node's version.
+Print version number.
```

Comment:
I did this change to bring it more in line Unix man page conventions. See, for example, the sample man page at http://linuxcommand.org/reading_man_pages.php where `--version` for `ls` says:

> output version information and exit

It does not say `output version information for ls and exit` or `output ls's version information and exit`.

Going into bikeshed mode for a moment, I do think that `version number` can be improved to `version information` as `6.2.0` is not actually a number and `version information` is a more accurate and informative description of what to expect.